src/Http/Livewire/LivewireDatatable.php

```diff
@@ -1063,7 +1067,7 @@ public function columnIsRelation($column)
 
     public function columnIsAggregateRelation($column)
     {
-        if (! $this->columnIsRelation($column)) {
+        if (!$this->columnIsRelation($column)) {
```

Comment:
@kountouris7 I changed those yesterday to have the same styling as the package, and to also make reviewing easier. Let's not introduce any styling changes with this PR.

Reply:
How did you changed them? Manually or is there any command?

Reply:
I did it manually in PHPStorm using search and replace